pkg/apis/kops/v1alpha2/componentconfig.go

```diff
@@ -160,6 +160,9 @@ type KubeProxyConfig struct {
 	Image string `json:"image,omitempty"`
 	// TODO: Better type ?
 	CPURequest string `json:"cpuRequest,omitempty"` // e.g. "20m"
+	CPULimit   string `json:"cpuLimit,omitempty"`
```

Comment:
Could we have some comments on the variables? .. just for consistency and linters